cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java

```diff
@@ -156,6 +161,46 @@ public void updateGradebook (@RequestBody GradebookDTO gradebook, @PathVariable(
 		
 	}
 	
+	@GetMapping("/assignment/{id}")
+	@Transactional
+	public Assignment getAssignment(@PathVariable int assignment_id) {
+		Assignment assignment = assignmentRepository.findById(assignment_id);
```

Comment:
I believe `.findById` returns `Optional<T>` type. I get a compilation error when I try to run the code.

Reply:
Made some modifications on my next commit